tests/rptest/tests/upgrade_test.py

```diff
@@ -614,3 +614,37 @@ def cloud_log_size() -> int:
         # Assert manifest has not been mutated by comparing sizes
         total_cloud_size_after_upgrade = cloud_log_size()
         assert total_cloud_size_before_upgrade <= total_cloud_size_after_upgrade, f"Mismatch in cloud storage size after upgrade, bytes before: {total_cloud_size_before_upgrade} bytes_after: {total_cloud_size_after_upgrade}"
+
+
+class InstallSelfTest(RedpandaTest):
+    @cluster(num_nodes=1)
+    def test_redpandainstaller_install(self):
+        """
+        Smoke test that checks RedpandaInstaller.install
+        :return:
+        """
+        install = lambda version: self.redpanda._installer.install(
+            self.redpanda.nodes, version)
+        start = lambda: self.redpanda.start(self.redpanda.nodes[0:1])
+        get_version = lambda: self.redpanda.get_version(self.redpanda.nodes[0])
+        previous_version = lambda version: self.redpanda._installer.highest_from_prior_feature_version(
+            version)
+
+        head_version, head_version_str = install(RedpandaInstaller.HEAD)
+        self.logger.info(f"base step: install(HEAD) -> {head_version_str}")
+        start()
```

Comment:
nit: probably don't need this since it's not testing the new two-tuple install path, and is redundant with `RedpandaTest.setUp()`

Reply:
I wanted to establish a base step to test the edge case of installing head and receiving the head_version.
I modified this part slightly to better reflect that